### ISSUE TITLE
The Loss on the Bow Model Goes Up

### ISSUE THREAD

**Issue by dspoka** [dmorr-google/wiki-reading#8]
I can't seem to figure out why the loss goes up. I'm training it on the shard of data you provided. 

Any thoughts?

**Comment by ilblackdragon** [comment]
You can try reducing learning rate. Also try adding `randomize_input` to `read_batch_features` call - currently code may be reading data sequentially.
